workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/WrappedTask.cs

```diff
@@ -21,6 +21,11 @@ public void Dispose()
             CancelSource?.Dispose();
         }
 
+        public int GetId()
+        {
+            return Task?.Id ?? 0;
```

Comment:
can 0 be a valid task id?

Reply:
I can't find clarification here: https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.id?view=netcore-2.2

But my understanding of this, 0 is not a valid task ID: https://blog.stephencleary.com/2013/03/a-few-words-on-taskid-and.html

Reply:
that sounds good to me